tests/drivers/aio/aio_basic_api/CMakeLists.txt

```diff
@@ -1,5 +0,0 @@
-include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
```

Comment:
Commit message states "tests: rename AIO api test" but I don't see a rename, it looks like the aio_basic_api test is simply being deleted in this commit. Maybe I'm missing something?

Reply:
fixed